lib/matplotlib/backends/backend_agg.py

```diff
@@ -265,14 +265,14 @@ def points_to_pixels(self, points):
         """
         return points * self.dpi / 72
 
-    def tostring_rgb(self):
-        return self._renderer.tostring_rgb()
+    def buffer_rgba(self):
+        return memoryview(self._renderer)
 
     def tostring_argb(self):
-        return self._renderer.tostring_argb()
+        return np.asarray(self._renderer).take([3, 0, 1, 2], axis=2).tobytes()
```

Comment:
Did you take a look at uses of this and ``tostring_rgb`` on things like StackOverflow to confirm that the type change would continue to work as expected in those use-cases?

Reply:
Here there is no change to the return type, it was a bytes, it's still a bytes.

There's a change in the return type of `buffer_rgba()` but that's sort of the whole point of the PR (and it goes back to the old Py2 behavior, and is also clearly better for performance).